apps/src/aichat/views/AichatView.tsx

```diff
@@ -115,6 +122,23 @@ const AichatView: React.FunctionComponent = () => {
     );
   }, [dispatch, initialSources, levelAichatSettings]);
 
+  useEffect(() => {
+    if (signInState === SignInState.SignedIn) {
+      try {
+        getUserHasAichatAccess().then(hasAccess =>
+          dispatch(setUserHasAichatAccess(hasAccess))
+        );
+      } catch (error) {
```

Comment:
Since this is using promise syntax and isn't an async function, the `try/catch` won't work as intended. Instead you can add a `catch` clause to the promise chain, i.e.
```
getUserHasAichatAccess()
  .then(...)
  .catch(error => ... handle error);
```

Reply:
Oh yeah! Forgot about that since we use `async` so much now. Will update.